.gitignore

```diff
@@ -5,9 +5,33 @@ dist/
 deploy/crds
 deploy/lighthouse/crds/lighthouse.submariner.io_*.yaml
 deploy/submariner/crds
+config/bundle/kustomization.yaml
 output/
 vendor/
 *.coverprofile
-.idea
 .shflags
 junit.xml
+
+# Binaries for programs and plugins
```

Comment:
Running many of the make targets locally, removing this .gitignore change, I don't see any modified files.

Imho, we should recommend folks configure their editors/IDEs to store their files in some location other than the repo. Generally, I think we should try to avoid getting into the business of ignores rules for dev-specific tools.

Reply:
I agree I'll remove this section

Reply:
I agree, I'll remove this section